### ISSUE TITLE
Loading Screen: Stuck in Space, erratic cursor

### ISSUE THREAD

**Issue by Nathaniel-MacIver** [tsunamayo/Skywanderers#891]
I LOVE YOUR GAME CONCEPT SO MUCH AND AM ROOTING FOR YA!

I have not been able to start a game since getting access to the experimental branch in steam 05/2018. When I start a new single player game, it loads for about 3 minutes, then stays at the starry sky. I can't use the left or right button, but when I click the middle mouse button, the cursor appears, and makes a clanking sound, almost like an item select or item load effect. Have to CTRL-ALT-DEL to exit the program.

...

1) Open the Game from Steam
2) Start a new single-player creative game
3) Create a new character (name: "me"), and proceed
4) Wait for the blinking loading text to stop blinking
5) Click the middle mouse button. The mouse will appear until released and make an 'item select' noise on doing so.

...

### Environment
- Skywanderers version: 2018.2.0b3 (0a6b93065060) (I think)
- Graphics Card: Intel(R) Iris(TM) Plus Graphics 640 (ID=0x5926) (Microsoft Surface Pro 2017)

### Additional information:

_(Screenshots, Videos, Error logs, Related issues)_

![after mouse click](https://user-images.githubusercontent.com/26629632/44623978-84770000-a8ad-11e8-8aaa-4917f3b6d931.png)

Please also include the `output_log.txt` file. You can find your log here: `%UserProfile%\AppData\LocalLow\TsunamayoCorp\Skywanderers\`

[output_log.txt](https://github.com/tsunamayo/Skywanderers/files/2321152/output_log.txt)
[SETTINGSini.txt](https://github.com/tsunamayo/Skywanderers/files/2321153/SETTINGSini.txt)

**Comment by Nathaniel-MacIver** [comment]
Oops, sry... Version 18w20a

**Comment by SelectedLime375** [comment]
I've had the same problem. I tried lowering the graphics settings, but the same thing always happens. I get the same output_log.txt as @Nathaniel-MacIver.

It seems to go wrong here:
D3D11: Failed to create RenderTexture (2736 x 1824 fmt 19 aa 1), error 0x887a0005

and then there are a bunch of "shader create error" and "failed to create buffer".

I'm on Windows 10, with Intel UHD Graphics 620, and DirectX 12.

**Comment by SelectedLime375** [comment]
Good news! I've found a way to get it working.

I went to AMD's page for my driver and downloaded the latest version of AMD Software. [Here](https://www.amd.com/en/support/graphics/radeon-500-series/radeon-500-series/radeon-530)'s the page where I got it from. I expanded "Windows 10 - 64-Bit Edition" and clicked the first "Download*" button. Once that setup, it brought of the software manager. If it isn't open, right-click on your desktop and select "AMD Radeon Settings." Then go to "System->Switchable Graphics." I then had to click "Running Applications->Installed Profiled Applications" (in the top-right). Then I browsed for the location of "Skywanders.exe" selected it and clicked "Open." I didn't see it added to the list, so I searched for it with the search bar and it came up. Then I clicked it and set it to "High Performance" Now I can load a game and place and break blocks.
